### DISCUSSION TITLE
Simple login

### DISCUSSION THREAD

**Discussion by shreyasajj** [dani-garcia/vaultwarden#2694] [General]
Hi bitwarden started supporting Simple Login: https://simplelogin.io/
ghacks.net/2022/06/02/bitwardens-username-generator-now-supports-simplelogin-anonaddy-and-firefox-relay-email-alias-services/

Is there any plans to integrated it into vaultwarden?

**Comment by BlackDex** [comment]
It already is supported in the latest version. This is all client side.
